### ISSUE TITLE
arepl is all screwed up on version 1.26.1

### ISSUE THREAD

**Issue by Almenon** [Almenon/AREPL-vscode#93]
Not sure why no-one has reported this yet. It was released over a week ago...

In version 1.26.1 for the first few runs the html is totally screwed up.  It looks like this:

> link rel=" hce/"3A/Ue="text/css"shc/f="file:///c%3A/Usess/aalmenon/.vsclme/pxt-nsions/.1menon.3rmpl-0d1.3amedia//ythonPreview.css"jsonRenderer.js"> scri t rcad>l///c%3A/Urs/a/.vs/extenson/mn.arepl-0.1.3/di/jsRener.j">scrit scri >> wiw.nlo=funco(){ rO ustpVaru/= {};="print">hello world
tjsonRndrer= rndrjs._co('+','-')//dfalicloka bi wird,vrr .her_s= w_ o_alval(2) tter.im/arepl/lobby">gitter | = .set_max_st g_leng; dcunt.getEeenByI("reslts").apndChd(jnRnderusrVar);} 
PrintOtut:helloworld 0 s

> brdiv id="foor" pmargn0x;repissu ⭐| talkongitter | Twet#arepl bdy>

After a few runs (usually just 2 but can be more) the html settles down and renders correctly.

**Comment by Almenon** [comment]
There must have been some breaking change in the v1.26.1 release... but I don't see anything significant in the changelog

https://github.com/Microsoft/vscode/milestone/76?closed=1

**Comment by Almenon** [comment]
It must have something to do with the timing of my html rendering- it renders correctly the first time and then quickly gets replaced by the gibberish.  I guess it doesn't like updating that fast?? 🤔 

I debugged it and it turns out my throttle code has a bug - it does a unnecessary recall after being called once.  I fixed it and the problem went away for the most part.

The problem still exists when there's lots of prints, but that is just a bug vs. a major bug.  I'll raise a issue on the VSCode repo.
